queries/omen/markets.ts

```diff
@@ -419,7 +427,30 @@ const getMarkets = async (
     fpmm => !marketHasDangerousKeyword(fpmm)
   );
 
-  return { fixedProductMarketMakers: filteredResults };
+  /**
+   * The Graph doesn't allow multiple values in the orderBy parameter (at least for now).
+   *
+   * This fn applies a secondary sort criterion if we get a primary sort result that
+   * has multiple markets with the same sorted value (for example, the same volume
+   * or same liq).
+   * This fn ONLY SORTS a given page results. (as we also apply a first and skip parameter
+   * values to the query result)
+   */
```

Comment:
```suggestion
  /**
   * The Graph doesn't allow to order by multiple fields.
   *
   * This fn applies a secondary sort criteria if we get a primary sort result that
   * has multiple markets with the same sorted value.
   * This fn only sorts a given page results. 
   */
```

What do you think about this? To be simpler

Reply:
Awesome! Thanks